src/frontend/src/main.tsx

```diff
@@ -1,7 +1,44 @@
+import { t } from '@lingui/macro';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
 import App from './App';
+import { HostList } from './states/states';
+
+// define settings
+declare global {
+  interface Window {
+    INVENTREE_SETTINGS: {
+      server_list: HostList;
+      default_server: string;
+      show_server_selector: boolean;
+    };
+  }
+}
+
+const IS_DEV = import.meta.env.DEV;
```

Comment:
@wolflu05 how do we enable this on Netlify while also building with production settings?

Reply:
Added a check via env variables. Let me know if that's ok for you.